<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnalyzeFun</class>
 <widget class="QWidget" name="AnalyzeFun">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>201</width>
     <height>321</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>功能栏</string>
   </property>
   <widget class="QPushButton" name="DataInputBtn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>数据导入</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="GraphBtn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>240</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>设定曲线</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PartAnalyzeBtn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>210</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>局部分析</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DataCatBtn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>180</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>数据查看</string>
    </property>
   </widget>
   <widget class="QPushButton" name="MarkBtn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>150</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>谐波标记</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AnalyzeBtn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>基频分析</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ArgsBtn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>参数设定</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PictureBtn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>图形绘制</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PackageBtn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>270</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>打包功能</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>340</y>
     <width>791</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>提示</string>
   </property>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>771</width>
      <height>81</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:12pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="StartBtn">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>460</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>开始分析</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>显示窗口</string>
   </property>
  </widget>
  <widget class="QPushButton" name="combkBtn">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>460</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
